### PR TITLE
board.is-an.app

### PR DESCRIPTION
**Pull request by Rahuletto**
# Verification Records

Type | Name | Value
-- | -- | --
TXT | _vercel | vc-domain-verify=board.is-an.app,f8602906e1897b3123cf

### Link to Website

Link: https://cdeboard.vercel.app

---

- [X] Toggle this checkbox to re-run the stargazer checking

Wanted: board.is-an.app
Thank you

### CODE REVIEW

File: domains/board.is-an.app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+addSubDomain({
+  description: 'An open-source code sharing platform that’s better in every way.',
+  domain: 'is-an.app',
+  subdomain: 'board',
+  owner: {
+    repo: 'https://github.com/Rahuletto/CodeBoard',
+    email: 'rahulmarban@gmail.com',
+  },
+  record: {
+    TXT: ['vc-domain-verify=board.is-an.app,f8602906e1897b3123cf'],
```
Comment (tarampampam) [inline domains/board.is-an.app.js:10]:
```suggestion
```

**Comment by Rahuletto** [comment]
I think its all good

**Comment by tarampampam** [comment]
Press the verification button now, please

**Comment by Rahuletto** [comment]
The domain works as intended. Thanks again ❤️

**Pull request merged**